ee/query-service/model/license.go

```diff
@@ -104,3 +104,53 @@ type SubscriptionServerResp struct {
 	Status string   `json:"status"`
 	Data   Licenses `json:"data"`
 }
+
+// ID and Key are the constant fields where as the entire entity can change based on decisions in control plane!
+type LicenseV3 struct {
+	ID   string                 `json:"id"`
+	Key  string                 `json:"key"`
+	Data map[string]interface{} `json:"data"`
+}
+
+type LicenseV3Aggreagate struct {
```

Comment:
Let's not call it Aggregate. There is no concept of aggregate in this entire codebase.

Reply:
```
func NewLicenseV3(data map[string]any) (*LicenseV3, error) { }
```